prisma/migrations/20250513202135_add_xmtp_installation_id_on_identities_on_devices/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[xmtpInstallationId]` on the table `IdentitiesOnDevice` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "IdentitiesOnDevice" ADD COLUMN     "xmtpInstallationId" TEXT;
```

Comment:
We are sure there are no duplicate values here?

Reply:
@neekolas Like that there could be 2 `xmtpInstallationId` with the same values for 2 different identity? Or identity on another device?

I think those are unique no? But you are in a better place to tell me

Reply:
Can you ever be registered twice without wiping the local DB. 

There’s one installation ID per local convos database

Reply:
Maybe my brain is dead but can you give an example of a scenario of what you are concern about?

The `xmtpInstallationId` is unique in the DB and it's linked to a unique `Device` and `DeviceIdentity` (which is what olds the inboxId

Reply:
If DeviceIdentity is as unique as the inboxID it should be fine. 

I don’t have enough context on when the device identities get created to have strong opinions